include/arch/arm/aarch64/scripts/linker.ld

```diff
@@ -222,7 +222,7 @@ SECTIONS
 #include <snippets-ram-sections.ld>
 
 #if defined(CONFIG_USERSPACE)
-#define APP_SHARED_ALIGN . = ALIGN(_region_min_align);
+#define APP_SHARED_ALIGN(size) . = ALIGN(_region_min_align);
```

Comment:
should this also be fixed here?

```
include/arch/arm/aarch32/cortex_a_r/scripts/linker.ld:#define APP_SHARED_ALIGN . = ALIGN(_region_min_align);
include/arch/arm/aarch32/cortex_m/scripts/linker.ld:#define APP_SHARED_ALIGN . = ALIGN(_region_min_align);
```

Reply:
Apparently the fix is not the only thing to be stupid, the author here just realized that pushed the wrong fix. I'm going to push the correct fix now (I'm supposed to fix `SMEM_PARTITION_ALIGN` not `APP_SHARED_ALIGN`). Anyway @nashif or the cortex_{ar,m} the fix is not needed because on both platform `SMEM_PARTITION_ALIGN` is defined to be the same as `MPU_ALIGN` that is taking the size parameter. On AArch64 `SMEM_PARTITION_ALIGN` is being defined as `MMU_ALIGN` that is not taking any parameter and that's the problem.